config/core/test-configs.yaml

```diff
@@ -1809,11 +1809,7 @@ test_configs:
       - baseline-nfs
       - cros-ec
       - igt-gpu-i915
-      - kselftest-filesystems
-      - kselftest-futex
       - kselftest-lib
-      - kselftest-lkdtm
-      - kselftest-seccomp
```

Comment:
We have a lot of x86 devices: 
10 [hp-11A-G6-EE-grunt](https://lava.collabora.co.uk/scheduler/device_type/hp-11A-G6-EE-grunt): filesystems, futex, lib, livepatch, lkdtm, seccomp
5 [asus-C433TA-AJ0005-rammus](https://lava.collabora.co.uk/scheduler/device_type/asus-C433TA-AJ0005-rammus): lkdtm, seccomp
5 [asus-C436FA-Flip-hatch](https://lava.collabora.co.uk/scheduler/device_type/asus-C436FA-Flip-hatch): filesystems, futex
5 [asus-C523NA-A20057-coral](https://lava.collabora.co.uk/scheduler/device_type/asus-C523NA-A20057-coral): lib and lkdtm
As discussed, we want to keep running one test on 2 devices at max of one architecture. So I'm keeping all 6 selftests enabled on grunt and distributing the remaining tests on the remaining three.

We can add more tests later to coral instead of running same tests on more than 2 devies. What do you think?

Reply:
I think it's more about running tests on 2 devices at least, not max. Some tests will provide different results on different platforms so it's also useful to cover them, such as different Intel CPU families. Having at least 2 just means we can see if the tests fail at the same time on both or not and to provide redundancy.  It depends on the kinds of tests, some are more likely to show differences than others.

Things like `lib` and `lkdtm` are probably unlikely to be hardware-specific, whereas `futex` and `seccomp` are more likely to differ at least across CPU architectures but also potentially within chip families. This has to be balanced with running more kselftests and other test suites. I think we'll have to keep adjusting things incrementally, probably on a weekly basis while keeping an eye on the load.

Reply:
Thank you for your explanation. I've updated the PR by removing changes for this board. We can keep iterating and adding more tests.